src/GovUk.Education.ExploreEducationStatistics.Content.Model/MethodologyParent.cs

```diff
@@ -1,9 +1,12 @@
-﻿using System;
+﻿﻿using System;
+using System.Collections.Generic;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.Model
 {
     public class MethodologyParent
     {
         public Guid Id { get; set; }
+        
+        public List<PublicationMethodology> Publications { get; set; }
```

Comment:
I think we should change ContentDbContext to configure this in this side of the relationship with something like `.WithMany(parent => parent.Publications)`

Reply:
Agreed!